Automattic/wp-calypso
# BBE Site Picker Step: remove confirmation modal

Fixes https://github.com/Automattic/martech/issues/2370

## Proposed Changes

* Removes the site delete confirmation modal on the BBE site picker step.

## Testing Instructions



* Go to `/start/do-it-for-me` and choose "Existing Site".
* On the site picker step, choose any available site.
* Confirm that you do not see any confirmation modal and proceed directly to the next step.
* Confirm that you are able to complete the purchase.

## Pre-merge Checklist
